Vagrantfile

```diff
@@ -81,7 +81,7 @@ Vagrant.configure("2") do |config|
      ./gtminstall --utf8 default
      gtmver=$(ls -1 /usr/lib/fis-gtm)
      echo "source /usr/lib/fis-gtm/$gtmver/gtmprofile" >> /home/ubuntu/.profile
-     ln -s $gtm_dist/libgtmshr.so /usr/local/lib
+     ln -s $gtm_dist/libgtmshr.so /vagrant/lib
```

Comment:
Ah ok. I'm not sure what "the linker" is (I'm somewhat new to Debian) but I'll take your word for it. Maybe there's a way to alter (or add to) the vagrant folders that are synced?